services.go

```diff
@@ -22,16 +24,28 @@ var _ mint.ServiceLimiter = (*staticServiceLimiter)(nil)
 
 // newStaticServiceLimiter instantiates a new static service limiter backed by
 // the given restrictions.
-func newStaticServiceLimiter(proxyServices []*proxy.Service) *staticServiceLimiter {
+func newStaticServiceLimiter(
+	proxyServices []*proxy.Service,
+) *staticServiceLimiter {
```

Comment:
I tried out `golines`, but unfortunately it doesn't produce the function signatures the way we want them. This should be formatted as:

```
func newStaticServiceLimiter(
	proxyServices []*proxy.Service) *staticServiceLimiter {
```

instead. Too bad there doesn't seem to be a formatting tool out there that we can configure that way...

Reply:
Gotcha. Yeah, tbh this is an issue with most formatters regardless of language in my experience. I guess it really comes down to a tradeoff between desired style vs. automated and reproducible standards. Line lengths are notoriously difficult to get "good" results (I imagine that's why `gofmt` left them off entirely).